client/state/ui/guided-tours/test/fixtures/config.js

```diff
@@ -21,7 +21,7 @@ export const MainTour = makeTour(
 );
 
 export const ThemesTour = makeTour(
-	<Tour name="themes" version="test" path="/design" when={ () => true } >
+	<Tour name="themes" version="test" path="/themes" when={ () => true } >
```

Comment:
Thanks for digging into this, apparently I wasn't seeing the forest for the trees. Fixed in 2294871